tests/Encryption/EncrypterTest.php

```diff
@@ -86,6 +86,10 @@ public function testOpenSslEncrypterCanDecryptMcryptedData()
             $this->markTestSkipped('Mcrypt module not installed');
         }
 
+        if (version_compare(PHP_VERSION, '7.1') > 0) {
+            $this->markTestSkipped('Not compatable with PHP 7.1+');
+        }
```

Comment:
You can still install the mcrypt extension on PHP 7.1 (infact, travis has), but running it fail the tests due to the deprecation.